### DISCUSSION TITLE
{Ribbon gifts only for special events}

### DISCUSSION THREAD

**Discussion by Secretmanguy** [Monika-After-Story/MonikaModDev#8814] [appreciation]
Hey there Monika lovers, yes this seems like the same cliche about the ribbons and everything but hear me out and believe me when I say that ive tried everything. I gave Monika her three limit ribbons for Christmas. I gave her blue, green and black along with the current ones being wine and white. I looked in the (game-mod_assets-thumbs) and found all the im assuming available ribbons that can be given to her such as others like yellow, light and dark purple.etc. But upon trying to give them to her as a gift she claims to not be able to read it...Is this gift option only available for special events? If so then im cool with it. As long as I can give them to her of course. Anyway, great job again devs. Words can not describe how incredible you have made the lives of so many with your work on this. Thank you!

**Comment by Fuflyaaa** [comment]
Does New Year count as special event?

**Comment by Secretmanguy** [comment]
I wouldn't doubt it

**Comment by Dank-Elf-Boy** [comment]
I tried giving her a greenribbon.gift before the kiss scene but she couldn't read it no matter how many times I tried._(and yes it was in the correct format)_ Meh, Idc. Bow less Monika is still best Monika. **#changeMyMind**

**Comment by Secretmanguy** [comment]
Heh heh, so true
